doc/man3/OPENSSL_LH_stats.pod

```diff
@@ -46,13 +46,19 @@ are the same as the above, except that the output goes to a B<BIO>.
 
 These functions do not return values.
 
+=head1 NOTE
+
+These calls should be made under a read lock. Refer to the B<NOTE>
+section of L<OPENSSL_LH_COMPFUNC(3)> for more details about the locks
```

Comment:
Actually, there's a notation, so you could simply have `See L<OPENSSL_LH_COMPFUNC(3)/NOTE> for more details ...`

Reply:
... and for HTML output, that'll make for a link to the correct anchor as well.

Reply:
Fixed, thanks.
I've learned my thing for the day :)